pages/filterCards/index.tsx

```diff
@@ -0,0 +1,91 @@
+import axios from "axios";
+// tslint:disable no-submodule-imports
+import { withRouter } from "next/router";
+import * as React from "react";
+import { RepoTags } from "../../typings/common";
+
+interface SelectedTags {
+  [tagName: string]: boolean;
+}
+
+interface Props {
+  data: {
+    tags: RepoTags;
+  };
+}
+
+interface State {
+  readonly selectedTags: SelectedTags;
+  tags: RepoTags;
+}
+
+class Index extends React.Component<Props, State> {
+  public state = {
+    selectedTags: ({} as any) as SelectedTags,
+    tags: ({} as any) as RepoTags
```

Comment:
without these `({} as any)` parts I get 
![image](https://user-images.githubusercontent.com/32563455/55320094-0ac87580-547f-11e9-9180-9256606df479.png)
As I understand it fixes also with `"no-object-literal-type-assertion": false` at `tslint.json` but I don't know is it good to use it

Reply:
direct `{} as SelectedTags` doesn't work?

Reply:
no I recieve this error:
```
Type assertion on object literals is forbidden, use a type annotation instead.
```

Reply:
No I get error:
```
Type assertion on object literals is forbidden, use a type annotation instead.
```
Is it ok to leave as it is?